package.json

```diff
@@ -20,10 +20,10 @@
     "jsdoc-to-markdown": "^4.0.0",
     "memory-fs": "^0.4.0",
     "postcss-import": "^11.0.0",
-    "postcss-js": "^1.0.0",
+    "postcss-js": "^2.0.0",
     "standard": "^11.0.0",
     "standard-version": "^4.0.0",
-    "sugarss": "^1.0.0",
+    "sugarss": "^1.0.1",
```

Comment:
Please leave it as `^1.0.0` since the `^` will already install the latest semver `minor` && `patch` within the `v1.x.x` range